corehq/apps/reports/standard/users/reports.py

```diff
@@ -48,8 +51,34 @@ def total_records(self):
 
     @memoized
     def _get_queryset(self):
-        # ToDo: filter based on params
-        query = LogEntry.objects
+        usernames = self._get_usernames()
+        # users needed to filter on, without it we would expose all log entries in the system
+        # so return empty if none available
+        if not usernames:
```

Comment:
LogEntry does not have "domain" available. So the only way to filter would be by the users.

Plus,
using usernames instead of user ids, because the user ID on the Log Entry is the django user ID and not couch user id available in user ES record.
The other way would be to fetch user ids by running a sql query on User for matching usernames, which would necessarily be the same, I believe.
I reckon this would run slow on prod, but this was okay to start with with usernames.